.travis.yml

```diff
@@ -0,0 +1,13 @@
+sudo: False
+language: python
+python:
+  - "3.5"
+  - "3.4"
+  - "2.7"
+  - "2.6"
```

Comment:
@mdrasmus Is supporting Python 2.6 desirable? I'm of the opinion that the world should move on, since 2.6 had hit end of life, and 2.7 introduced a lot of nice sugar like dictionary comprehensions and set literals.